README.markdown

```diff
@@ -24,6 +24,8 @@ Or even better, use [pathogen.vim][1] and simply pull it in like this:
 
     cd ~/.vim/bundle
     git clone https://github.com/PProvost/vim-ps1.git
+    
+If you use [vim-plug][4] add `Plug 'pprovost/vim-ps1'` to your config.
```

Comment:
I like the idea, but please follow the same block style as the example above this for using pathogen. Makes it easier to see what users should do at a glance.

Reply:
Agreed. I changed it to use the same 4 space indentation block as the pathogen instructions.